libs/wallet/src/connect-dialog/connect-dialog.spec.tsx

```diff
@@ -117,7 +150,7 @@ it('Unsuccessful connection using rest auth form', async () => {
     fireEvent.submit(screen.getByTestId('rest-connector-form'));
   });
 
-  expect(spy).toHaveBeenCalledWith(fields);
+  expect(spy).toHaveBeenCalledWith('http://localhost:1789/api/v1', fields);
```

Comment:
Chore (non-blocking): this url's used a few times, maybe nicer to put in a var

Reply:
will put in a follow up, want to get this merged